src/rxn_aa_mapper/model.py

```diff
@@ -25,17 +76,32 @@ def __init__(
         """
         super().__init__()
         self.model_args = model_args
+        self.model_architecture = model_args.get("architecture", {})
+
         resume = self.model_args.get("resume-training", False)
         if resume:
             logger.info(
                 f"resume the training from the checkpoint : {self.model_args['model']}"
             )
             self.model = AutoModelForMaskedLM.from_pretrained(model_args["model"])
         else:
-            self.config = AutoConfig.from_pretrained(
-                model_args["model"], **model_architecture
-            )
-            self.model = AutoModelForMaskedLM.from_config(self.config)
+            if "albert" == model_args["model"].split("_")[-1]:
```

Comment:
what about a `startswith` and no split?

Reply:
In this case it may not work, because if we use models from protTrans, they are written like: prot_bert or prot_albert

Reply:
resolved:
* check if Albert in name, then it's Albert Model 
* check if Bert is in name but not Albert, then its Albert. 
Without doing splits...